integration-tests/scenarios/user_management.md

```diff
@@ -0,0 +1,6 @@
+# Feature: User Management API
+- Scenario: Get UUID via post call
+- Scenario: Trying out the API without user key
```

Comment:
can you add a check for the user status of REGISTERED and FREE TIER instead of just checking the response code, I will figure out how to do it for expired.

Reply:
done